static/css/styles.css

```
@@ -1160,6 +1218,74 @@ table .text-end { text-align: right; }
+  border-radius: var(--radius-md);
+  background: hsla(42, 90%, 55%, 0.08);
+  color: var(--text-secondary);
+}
+
+.task-schedule-notice strong {
+  color: var(--warning-text);
+}
+
+.task-schedule-notice code {
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)